elfpy/pricing_models/hyperdrive.py

```diff
@@ -107,17 +106,18 @@ def calc_in_given_out(
             The result of performing the trade
         """
 
+        # pylint: disable=too-many-locals
+
         # Calculate some common values up front
         out_amount = Decimal(out.amount)
-        normalized_time = Decimal(time_remaining.normalized_time)
+        curve_portion = Decimal(time_remaining.normalized_time)
```

Comment:
I find these variable names to be more confusing because it no indicates that it is an indicator of time until maturity. What about switching `normalized_time` to `time_remaining` and leaving `(1-time_remaining)`?

Reply:
done in f10f1e770d8559883fdf267802c77f0b6a9ef96e